core/src/main/java/com/datastax/oss/driver/internal/core/adminrequest/AdminRequestHandler.java

```diff
@@ -110,12 +110,12 @@ private void onWriteComplete(Future<? super Void> future) {
           channel.eventLoop().schedule(this::fireTimeout, timeout.toNanos(), TimeUnit.NANOSECONDS);
       timeoutFuture.addListener(UncaughtExceptions::log);
     } else {
-      result.completeExceptionally(future.cause());
+      setFinalError(future.cause());
     }
   }
 
   private void fireTimeout() {
-    result.completeExceptionally(
+    setFinalError(
```

Comment:
Does the time a request spend throttled factor into the client timeout?  From what I can tell I think it is.  I think depending on how you are using the throttle (i.e. hammering hard and using driver to limit your throughput), it's like a user would also want to raise the client timeout.   I think in documentation we should probably want to make that clear that the time a request spends throttled factors in client timeout.

Reply:
Yes, it does factor into the client timeout. I think it's mentioned somewhere in the configuration comments, I'll make sure to explain it in the manual too.